cmd/bucket-replication.go

```diff
@@ -1425,6 +1425,7 @@ func (p *ReplicationPool) queueReplicaFailedTask(ri ReplicateObjectInfo) {
 		})
 	case p.mrfReplicaCh <- ri:
 	default:
+		logger.LogOnceIf(GlobalContext, fmt.Errorf("WARNING: Replication failed workers could not keep up with healing failures - consider increasing number of replication failed workers with `mc admin config set api replication_failed_workers=`"), replicationSubsystem)
```

Comment:
perhaps even suggest a value based on the current value? @poornas

Reply:
updated the PR.